samples-generator/bin/3d-tiles-samples-generator.js

```diff
@@ -1,5 +1,6 @@
 #!/usr/bin/env node
```

Comment:
If running in non-legacy mode don't write samples for `BatchedDeprecated1` and `BatchedDeprecated2`. This is way legacy that the 3D Tiles Next doesn't have to worry about.